### DISCUSSION TITLE
Barcode scanner not working

### DISCUSSION THREAD

**Discussion by prasanthmarimi** [ionic-team/capacitor#5150] [Q&A]
Hi, trying to implement the barcode scanner in my vue.js project but it's getting me into runtime errors. Could anyone please suggest any solution or alternative way of using it? Thanks.

**Comment by jcesarmobile** [comment]
If you provide more information about what plugin you are using, what runtime errors are you getting, etc. maybe we can help you, with that little information is impossible to give you an answer.
